third_party/terraform/tests/resource_storage_hmac_key_test.go

```diff
@@ -0,0 +1,68 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
```

Comment:
I think we need the `github.com/hashicorp/terraform-plugin-sdk/helper/acctest` package as well.